### PR TITLE
MAGN-6760 Change name from ColoredGeometry to Display.ByGeometryColor

### PR DESCRIPTION
**Pull request by ikeough**
This PR changes the name of `ColoredGeometry.ByGeometryAndColor` to `Display.ByGeometryColor`.

PTAL:
@pboyer

### CODE REVIEW

**Comment by pboyer** [comment]
@kronz @ikeough @lillismith  This is really weird.  This class is named after a verb?  By that same reasoning, shouldn't `Sphere` be called `MakeSphere`?  The coloration is not a mutation of the original object, but creates a new object!  Maybe it should be called a `Preview`?

**Comment by lillismith** [comment]
@pboyer @kronz @ikeough How about Coloration.ByGeometryColor?

**Comment by pboyer** [comment]
Part of the reason by this doesn't work for me is that colors are not the only thing enabled by this work.  What about materials? What about shaders?  Real time rendering?  How will that gracefully fit in?  This is badass shit and we should not miss the boat.  ;)

I prefer, `Preview.ByGeometryColor` because the simple fact is that color is just one of many things enabled by this work.  Or , `PreviewGeometry.ByGeometryColor`?  

Preview is a nice word because it is both a verb and a noun, so works with both approaches.

**Comment by pboyer** [comment]
@lillismith and I had a discussion and decided on a few different class names that could make sense:

1. `Display`
2. `Visualization`
3. `Appearance`

Static constructors:

`.ByGeometryColor`
`.ByGeometryMaterial`
`.ByGeometryShader`

E.g.  `Display.ByGeometryColor`   ->  Yield a `Display` object

We agreed that these anticipate the more general capabilities of the rendering pipeline.  @ikeough @kronz What do you think?

**Comment by kronz** [comment]
@ikeough @pboyer @lillismith Display. does solve a number of issues quite nicely.  There was a lot of ambiguity about what the output of this thing was, something that GH solves by making these weirdo cutoff nodes, and that we were not addressing head on.  I like that it yield a new element.  It's a little weird for now in terms of category, where does it live in the UI?  So I think we should go with Display, it's expressive, accurate, clear, and short.

**Comment by lillismith** [comment]
agreed Display.ByGeometryColor

**Comment by jnealb** [comment]
Ditto

**Comment by pboyer** [comment]
Closing until this is updated.

File: src/Libraries/CoreNodes/GeometryColor.cs [hunk 1]
```diff
@@ -9,18 +9,24 @@
...
             this.color = color;
         }
 
-        public static ColoredGeometry ByGeometryAndColor(Geometry geometry, Color color)
+        /// <summary>
+        /// Display geometry using a color.
+        /// </summary>
+        /// <param name="geometry">The geometry to which you would like to apply color.</param>
+        /// <param name="color">The color.</param>
+        /// <returns></returns>
```
Comment (pboyer) [inline src/Libraries/CoreNodes/GeometryColor.cs]: No reason to have an empty `returns` XML element.

**Comment by pboyer** [comment]
LGTM except for a minor quibble.

**Pull request merged**